qemu/tests/boot_from_device.py

```diff
@@ -107,10 +109,9 @@ def check_boot_result(boot_fail_info, device_name):
             except ValueError:
                 output = vm.serial_console.get_output()
 
-            for i in infos:
-                if not re.search(i, output):
-                    raise error.TestFail("Could not boot from"
-                                         " '%s'" % device_name)
+            if re.search(infos[1], output):
```

Comment:
Can you help split `info[0]` and `info[1]` into two different parameters in the configuration file? It is misleading because only `info[1]` is the real `boot_fail_info`.

Reply:
agree with luckyh, @aliang123 please check it.